### PR TITLE
Fix typos in Russian translation

### PR DESCRIPTION
**Pull request by lex111**
This pretty large PR contains simple typos fixes in Russian texts, performed in semiautomatic mode (using Yandex Speller with custom dictionary of exceptions). I think this is good first step to facilitate subsequent review of all articles and to reduce the bug reports with typos from users.

I have reviewed all the replacements several times, so most of them (99%) are safe to apply. But it was great if someone from @mdn/yari-content-ru examine these changes.

NB: This PR does not fix mistranslation except in some cases, since I plan to do so in my next PR.

### CODE REVIEW

**Comment by SphinxKnight** [comment]
I think it could be useful, for a first PR to Russian content, to split this into several "chunks" (cf. item 7 of https://github.com/mdn/content#more-substantial-changes)

**Comment by lex111** [comment]
This PR is hard to divide into several "chunks", since it contains related set of changes, so I don't see the problem with merge it.
It's initial work on ru content review to make it easier for the team to work with it, without getting distracted by fixing simple typos.

**Review by chrisdavidmills — APPROVED** [review]
I'll take the responsibility to merge this, so work can move forward.

In future @lex111 , it would be good if you could break the work up a bit if possible, as it is really hard to review such a PR. Maybe run your script on a certain subtree at a time?

**Comment by lex111** [comment]
@chrisdavidmills thanks, alright, maybe that makes sense. I will prepare another PR with mass-edits, which will split into two separate PRs.

But is there more concrete criterion for determining when whether PR should be splitted into several chunks? How many changed files/lines for example?

File: files/ru/web/progressive_web_apps/installable_pwas/index.html [hunk 1]
```diff
@@ -94,7 +94,7 @@ <h2 id="Добавить_на_домашний_экран">Добавить на
 
 <p><img alt="" src="https://mdn.mozillademos.org/files/15834/js13kpwa-installed.png" style="border-style: solid; border-width: 1px; display: block; margin: 0px auto;"></p>
 
-<p>После этого пользователь может запустить приложение и сразу начать его использовать. Обратите внимание, иногда у PWA (в зависимости от браузера и/или операционной системы) в правом нижнем углу значка отображется иконка браузера, чтобы инормировать пользователя о его веб-характере.</p>
+<p>После этого пользователь может запустить приложение и сразу начать его использовать. Обратите внимание, иногда у PWA (в зависимости от браузера и/или операционной системы) в правом нижнем углу значка отображется иконка браузера, чтобы игнорировать пользователя о его веб-характере.</p>
```
Comment (captainspring) [inline files/ru/web/progressive_web_apps/installable_pwas/index.html]: Точно «игнорировать», а не «информировать»?

**Review by yanaklose — COMMENTED** [review]
Caught a few little typos that still remained.
Поймала несколько мелких ошибок.

**Comment by chrisdavidmills** [comment]
@lex111 Probably 500 per time? I tend to review these by just looking at about every tenth file, so I can get an idea of what the changes are on average.

@captainspring I am really sorry — I did not realize you were working on this. I wonder can we address your feedback in a separate PR?

**Comment by captainspring** [comment]
@chrisdavidmills no problem, separate PR is ok. It’s just a few little typos, nothing too crucial.

**Pull request merged**